README.md

```diff
@@ -20,7 +12,12 @@ All documentation is available online:
 
 ## Overview
 
-This repository is used to build the client-side cross-platform Telemetry library part of [Glean](https://docs.telemetry.mozilla.org/concepts/glean/glean.html), called the `Glean SDK`.
+There currently two independent implementations of the Glean SDK:
```

Comment:
I would put this at the end of this section, rather than mentioning it upfront

Reply:
I had considered that, but then the subsequent paragraphs don't have any context.  I've reorganized this more substantially.  Let me know what you think.

Reply:
Thank you, looks good to me now!